Scripts/CheckEmailAuthenticity/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
```

Comment:
@kirbles19 those are the changes made here:
```
  - Updated descriptions of command arguments and outputs.
  - output paths will always be set
```

Reply:
Output paths are always set (populated).

Not sure what "set" means...

Reply:
@kirbles19 "populated" is good

Reply:
@orlichter1 please fix the CHANGELOG accordingly